app/utility/transforms/CollapsedMetrics.js

```diff
@@ -64,7 +64,7 @@ export const getAvgROM = (set) => {
 
```

Comment:
- [ ]  Ideally we make the dividing of duration into a separate convenience class. F,or now, just making it a trello card, though can also fix it here